### PR TITLE
Rubocop: Address RSpec/ContextWording exclusions in CCMS tests

### PR DESCRIPTION
**Pull request by colinbruce**
## What

Start addressing the context exclusions in the CCMS namespace, remove another 106 infractions

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: spec/services/ccms/attribute_value_generator_spec.rb [hunk 1]
```diff
@@ -66,25 +66,25 @@ module CCMS
     end
 
     describe "#respond_to?" do
-      context "standardly_named methods" do
+      context "with a standardly_named methods" do
```
Comment (agoldstone93) [inline spec/services/ccms/attribute_value_generator_spec.rb:69]:
```suggestion
      context "with standardly_named methods" do
```

Just a little typo

File: spec/services/ccms/manual_review_determiner_spec.rb [hunk 2]
```diff
@@ -62,31 +62,31 @@ module CCMS
...
                 expect(subject).to be true
               end
             end
 
-            context "no contribution" do
+            context "and there is no contribution" do
               let!(:cfe_result) { create(:cfe_v3_result, submission: cfe_submission) }
 
-              context "restrictions" do
+              context "and there are restrictions" do # TODO: check this - description does not match expectation
```
Comment (agoldstone93) [inline spec/services/ccms/manual_review_determiner_spec.rb:83]: Did you mean to leave this TODO in?

File: spec/services/ccms/manual_review_determiner_spec.rb [hunk 3]
```diff
@@ -130,27 +130,27 @@ module CCMS
...
                 expect(subject).to be true
               end
             end
 
-            context "no contribution" do
+            context "when there is no contribution" do
               let!(:cfe_result) { create(:cfe_v3_result, submission: cfe_submission) }
 
-              context "restrictions" do
+              context "and there are restrictions" do # TODO: check this - description does not match expectation
```
Comment (agoldstone93) [inline spec/services/ccms/manual_review_determiner_spec.rb:147]: is this also meant to be left in?

File: spec/services/ccms/manual_review_determiner_spec.rb [hunk 4]
```diff
@@ -160,27 +160,27 @@ module CCMS
...
                 expect(subject).to be true
               end
             end
 
-            context "no contribution" do
+            context "and there is no contribution" do
               let!(:cfe_result) { create(:cfe_v3_result, submission: cfe_submission) }
 
-              context "restrictions" do
+              context "and there are restrictions" do # TODO: check this - description does not match expectation
```
Comment (agoldstone93) [inline spec/services/ccms/manual_review_determiner_spec.rb:177]: Same again

File: spec/services/ccms/parsers/case_add_response_parser_spec.rb [hunk 5]
```diff
@@ -39,7 +39,7 @@ module Parsers
         end
       end
 
-      context "unsuccessful response" do
+      context "when the response is sunsuccessful" do
```
Comment (agoldstone93) [inline spec/services/ccms/parsers/case_add_response_parser_spec.rb:42]:
```suggestion
      context "when the response is unsuccessful" do
```
Another little typo

File: spec/services/ccms/parsers/reference_data_response_parser_spec.rb [hunk 6]
```diff
@@ -6,7 +6,7 @@ module Parsers
       let(:expected_tx_id) { "20190301030405123456" }
       let(:expected_reference_number) { "300000135140" }
 
-      context "successful response" do
+      context "when the successful is response" do
```
Comment (agoldstone93) [inline spec/services/ccms/parsers/reference_data_response_parser_spec.rb:9]: Should this say 'when the response is successful'?

**Review by agoldstone93 — APPROVED** [review]
Looks good to me. There's a couple of typos and I just wanted to check if you left in those TODOs on purpose but otherwise all good

**Comment by colinbruce** [comment]
The todos were deliberate as I don't want to get distracted with fixing _all_ the issues in the space and I want to dig into the history and find out why someone either wrote or changed the test to expect the opposite of the description!

I'll take a look at the typos and push up fixes :+1:

**Pull request merged**